src/Streamable.js

```diff
@@ -0,0 +1,8 @@
+import fetch from 'node-fetch';
+
+export const shouldTransform = url => /^https?:\/\/streamable\.com\//.test(url);
+
+export const getHTML = url =>
+  fetch(`https://api.streamable.com/oembed.json?url=${url}`).then(res =>
+    res.json().then(streamable => streamable.html)
```

Comment:
Just return `res.json` and add the `then` on the `fetch`

```js
fetch(`https://api.streamable.com/oembed.json?url=${url}`)
  .then(res => res.json())
  .then(({ html }) => html);
```

Even tho we don't need to do a fetch call
We can just return the `iframe` ourselves and save us a fetch call
```html
<iframe src="https://streamable.com/o/moo" frameborder="0" scrolling="no" width="100%" height="480" allowfullscreen></iframe>
```

Reply:
What your opinion on this one (fetch vs. doing it ourselves) @kentcdodds? 🤔

Reply:
If they ever change the code that's required to embed then doing the fetch call will be a good thing, so I say let's do that.